framework/yii.bash-completion

```diff
@@ -0,0 +1,31 @@
+# This file implements bash completion for the ./yii command file.
+# It completes the commands available by the ./yii command.
+# See also:
+# - https://debian-administration.org/article/317/An_introduction_to_bash_completion_part_2 on how this works.
+# - https://www.gnu.org/software/bash/manual/html_node/Programmable-Completion.html
+#
+# Usage:
+# Temporarily you can source this file in you bash by typing: . yii.bash-completion
```

Comment:
I'd prefer `source yii.bash-completion` instead of `. yii.bash-completion` as it is more readable and less confusing.